pkg/monitoring/metrics/virt-controller/metrics.go

```diff
@@ -40,19 +40,23 @@ var (
 		vmiMetrics,
 	}
 
-	vmInformer                  cache.SharedIndexInformer
-	vmiInformer                 cache.SharedIndexInformer
-	clusterInstanceTypeInformer cache.SharedIndexInformer
-	instanceTypeInformer        cache.SharedIndexInformer
-	clusterPreferenceInformer   cache.SharedIndexInformer
-	preferenceInformer          cache.SharedIndexInformer
-	vmiMigrationInformer        cache.SharedIndexInformer
-	clusterConfig               *virtconfig.ClusterConfig
+	vmInformer                    cache.SharedIndexInformer
+	vmiInformer                   cache.SharedIndexInformer
+	persistentVolumeClaimInformer cache.SharedIndexInformer
+	dataVolumeInformer            cache.SharedIndexInformer
+	clusterInstanceTypeInformer   cache.SharedIndexInformer
+	instanceTypeInformer          cache.SharedIndexInformer
+	clusterPreferenceInformer     cache.SharedIndexInformer
+	preferenceInformer            cache.SharedIndexInformer
+	vmiMigrationInformer          cache.SharedIndexInformer
+	clusterConfig                 *virtconfig.ClusterConfig
 )
 
 func SetupMetrics(
 	vm cache.SharedIndexInformer,
 	vmi cache.SharedIndexInformer,
+	pvc cache.SharedIndexInformer,
+	dataVolume cache.SharedIndexInformer,
```

Comment:
you need to add the new informer args in `pkg/virt-controller/watch/application.go:435`